readthedocs/config/tests/test_config.py

```diff
@@ -222,15 +222,15 @@ def test_python_pip_install_default():
     assert build.pip_install is False
 
 
-def describe_validate_python_extra_requirements():
+class ValidatePythonExtraRequirements:
```

Comment:
We need to follow the convention of naming the class starting with `Test` like `TestValidatePythonExtraRequirements` and inheritance from object to keep the code python2 compatible.